internal/external-dns/registry/txt.go

```diff
@@ -39,6 +42,8 @@ const (
 
 	txtFormatVersion         = "1"
 	externalDNSMapperVersion = ""
+
+	GroupLabelKey = "group"
```

Comment:
We [have](https://github.com/philbrookes/dns-operator/blob/gh-620/types/group.go#L10) this key in the `types/group.go`